documentation/docs/00-introduction.md

```diff
@@ -2,25 +2,54 @@
 title: Introduction
 ---
 
-### Before we begin
+SvelteKit is an application framework for building web apps using Svelte components.
 
-> SvelteKit is in early development, and some things may change before we hit version 1.0. This document is a work-in-progress. If you get stuck, reach out for help in the [Discord chatroom](https://svelte.dev/chat).
->
-> See the [migration guides](/migrating) for help upgrading from Sapper.
+Build modern state-of-the-art web apps that fully embrace the serverless paradigm with a great development experience.
+
+Under the hood SvelteKit is made up of multiple things, hence the name "kit". It's a rendering engine for SSR and CSR, a client-side runtime for client-side routing and CSR, and a toolchain for development.
+
+(If you need a quick reference on these words, check out the [glossary](#appendix) in the appendix.)
+
+### Features
+
+- [server-side rendering](#appendix-ssr)
+
+For the initial page, the user is served a server-side rendered version of the page.
```

Comment:
I'm not 100% sure without testing it, but I think that these bullet points will not render correctly. This looks to me like it will create a series of one bullet point lists interleaved with new paragraphs. I think the description needs to be on the same line as the bullet point

Reply:
I'm not sure, because the docs are now loaded from an API. @pngwn do you have ideas on testing docs rendering?

Reply:
I started work on this and dropped the ball a bit, so we could test locally. For now we'll just have to make a best judgement and go from there. I'll finish off the test setup I had in mind this weekend but it will probably need publishing for it to be useful.